oic.r.svc.raml

```diff
@@ -10,7 +10,7 @@ traits:
         if:
             enum: ["oic.if.baseline"]
 
-/SecSvcResURI:
```

Comment:
The /oic/sec/svc Resource is removed from OCF 1.0, and since it wasn't mandatory in OIC 1.1, I'm not sure OCF OBTs are required to comprehend it.  Are we keeping it around for completeness sake, or ???